package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "FrontCamp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --config ./webpack.config.js --mode development",
+    "build": "webpack --mode production",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d dist",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint src/**/*.js",
```

Comment:
Do you run the linter?

Reply:
![image](https://user-images.githubusercontent.com/43151213/68075891-0e2e3180-fdbf-11e9-9b44-5ae822510c72.png)
It's working